### DISCUSSION TITLE
Issue with POE coordinator - after major power outage no connection

### DISCUSSION THREAD

**Discussion by kelsonengineering** [tube0013/tube_gateways#149] [Q&A]
Hello,

We've been happily using the POE coordinator with HA for several months. Last week, we had a major storm that knocked power out for a large area over several states. Now, the device is not working - the red light turns on (power, I presume), but the device does not connect to my router. We checked our OPNsense router and it is not connected.

Does any one have advice for how I can troubleshoot this? Is there a way to read messages during boot?

Thanks so much.
Vic

**Comment by tube0013** [comment]
you can connect the device to usb via the internal USB port and use UNPOWERED ethernet. to watch the console.

I've seen a few instances where power outages like this have knocked out the olimex esp32-poe module with exactly as you describe.

Let me know if you get anything on the serial console when doing usb power, otherwise contact me at store@tubeszb.com and can work out a replacement module.

**Reply by kelsonengineering** [reply]
I didn't get anything on the serial console. Alas!
